crates/sui-json-rpc-types/src/sui_move.rs

```diff
@@ -91,19 +91,25 @@ pub enum SuiMoveNormalizedType {
     U256,
     Address,
     Signer,
-    #[serde(rename_all = "camelCase")]
     Struct {
-        address: String,
-        module: String,
-        name: String,
-        type_arguments: Vec<SuiMoveNormalizedType>,
+        #[serde(flatten)]
+        inner: Box<SuiMoveNormalizedStructType>,
```

Comment:
I wanted to fix the `enum` size here, this was the only way I could figure out to do so without changing the json layout